js/components/controlBar.js

```diff
@@ -143,17 +148,19 @@ var ControlBar = React.createClass({
    * @private
    */
   setLandscapeScreenOrientation: function() {
-    var orientation = window.screen.orientation || window.screen.mozOrientation || window.screen.msOrientation;
-    if (orientation && orientation.type && (orientation.type === "portrait-secondary" || orientation.type === "portrait-primary")) {
-      var orientations = "landscape-primary";
-      if (screen.orientation && screen.orientation.lock) { //chrome browser
-        screen.orientation.lock(orientations);
-      } else if (screen.lockOrientation) { //new one
-        screen.lockOrientation(orientations);
-      } else if (screen.mozLockOrientation) { //ff
-        screen.mozLockOrientation(orientations);
-      } else if (screen.msLockOrientation) { //ie
-        screen.msLockOrientation(orientations);
+    if (this.props.controller && this.props.controller.checkDeviceOrientation) {
+      var orientation = window.screen.orientation || window.screen.mozOrientation || window.screen.msOrientation;
+      if (orientation && orientation.type && (orientation.type === "portrait-secondary" || orientation.type === "portrait-primary")) {
```

Comment:
Should this be using `Utils.getOrientationType()`?

Reply:
\+